### DISCUSSION TITLE
Hantek 6022BE vs 6022BL?

### DISCUSSION THREAD

**Discussion by DeeKey** [OpenHantek/OpenHantek6022#235] [Q&A] [answered]
Is there any advantage of buying version 6022BL with logic analyzer over 6022BE (which is just a oscilloscope)?  
The problem is that logic analyzer and oscilloscope cannot work at the same time in 6022BL model. 
So it is better to buy cheap stand alone logic analyze (like https://www.ebay.de/itm/EEPROM-CY7C68013A-56-EZ-USB-FX2LP-USB2-0-Develope-Board-Module-Logic-Analyzer/232475320246 or similar) and  6022BE model which is a bit cheaper.

Am I right or there are still some advantages in BL model?

**Comment by Vascom** [comment]
I have 6022BL and not see advantages because OpenHantek6022 not supported logic analyzer.

**Reply by bill-in-iowa** [reply]
I have the 6022bl, I bought a good scope and just use the logic analyzer in the 6022

**Comment by DeeKey** [comment, marked as answer]
Well, it is supported by Sigrok as LogicAnalyzer. But still what is the point in it, as it is better (and cheaper even) to buy simple stand alone logic analyzer...

https://sigrok.org/wiki/Hantek_6022BL

**Reply by area-8051** [reply]
I second this. The 6022BL has a 16-channel logic analyser, but advertises itself as an 8-channel Salaea Logic, so Sigrok lets you use only the first 8 channels. And you have to select the 6022BL mode with a switch on the rear panel, then turn it off and on.
In the end, you're much better off with 2 separate devices.

**Reply by MaMiKoe** [reply]
I own a 6022BL and would buy the next time the 6022BE. I can/could use it with the Saleae-Software and driver als 8-channel-LA, but I assume when needing this 8 channels are to less; it would be better buying additional a cheap&simple saleae-clone with 16 channels. And I assume the problems I have with using the Software PCScope result from having the BL and not the BE.

**Reply by Ho-Ro** [reply]
Yes, it is a shame that Hantek have not made these two very similar units HW compatible. Only a few port lines are assigned differently, which control the input amplifiers and the LEDs. With the LE, only one line has been added for the analogue/LA switchover.
